src/features/pastelDB/pastelDBLib.ts

```diff
@@ -0,0 +1,80 @@
+import fs from 'fs-extra'
+import path from 'path'
+import { Database } from 'sql.js'
+
+import {
+  create_block,
+  create_blocksubsidy,
+  create_chaintips,
+  create_listaddresses,
+  create_listreceivedbyaddress,
+  create_listtransactions,
+  create_listunspent,
+  create_mempoolinfo,
+  create_mininginfo,
+  create_nettotals,
+  create_networkinfo,
+  create_rawmempoolinfo,
+  create_rawtransaction,
+  create_statisticinfo,
+  create_totalbalance,
+  create_transaction,
+  create_txoutsetinfo,
+  create_walletinfo,
+} from './constants'
+
+const initSqlJs = require('./sql-wasm.js')
+
+export const createDatabase = async (): Promise<Database> => {
+  const SQL = await initSqlJs({
+    locateFile: (file: string) => {
+      return `static/bin/${file}`
+    },
+  })
+
+  let filebuffer
+  try {
+    filebuffer = fs.readFileSync(
+      path.join(process.env.HOMEPATH as string, './pasteldb.sqlite'),
```

Comment:
Can we move this to a separated file, like src/config.ts?
Also there would be a APP_DATA path of this app already, you should use it instead. For eg: see src/legacy/components/LoadingScreen.tsx